sp-scheduler.py

```diff
@@ -0,0 +1,45 @@
+import time
+import random
+import json
+import numpy as np
+import kubernetes.client
+
+from kubernetes import client, config, watch
+from heuristics import spot_over_non_spot_always, least_volatile_nodes_always
+
+config.load_kube_config('./config-spark-on-cat')
+v1=client.CoreV1Api()
+
+scheduler_name = "spot-scheduler"
+
+def get_volatility_metrics(instance_name):
```

Comment:
not needed?

Reply:
scheduler_name is because I perform a string pattern match in main, (see line #33 in sp-scheduler.py)

This can be passed as a command line param during startup however.